beakerx_tabledisplay/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@beakerx/beakerx-tabledisplay",
-  "version": "2.4.3",
+  "version": "2.4.4",
```

Comment:
Versioning ergonomics tangent:

This would be for the front-end which I don't think is strictly required here for this patch? If this is to keep the two versions of the widget pinned together, that seems preferred to me too. Is there a versioning mechanism that other Jupyter projects use in these scenarios, where they can bump both versions without updating the respective lock/requirements files individually?